### PR TITLE
added websocket file for lisibility and updated websocket to parse return from API 

### PR DESCRIPTION
**Pull request by troux**
https://craft-ai.atlassian.net/browse/CRAI-3913
Added a new file for websocket function for lisibility 
Change the return management of websocket to reflect change in API

### CODE REVIEW

File: tests/test_add_agent_operations.py [hunk 1]
```diff
@@ -93,6 +94,7 @@ def test_add_agent_operations_with_some_duplicates(self):
         self.assertEqual(resp["added_operations_count"], len(operations) - 2)
 
 
+@unittest.skip
```
Comment (matthieu-boussard) [inline tests/test_add_agent_operations.py:97]: skip ?

File: tests/test_add_agent_operations.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 from .data import valid_data, invalid_data
 
 
+@unittest.skip
```
Comment (matthieu-boussard) [inline tests/test_add_agent_operations.py:16]: why skipping this test ? Please remove the test or activate it or comment why skipped

File: tests/test_add_agent_operations.py (hunk 1 above)
Comment (troux) [reply tests/test_add_agent_operations.py:97]: woops, reminder from testing

**Comment by matthieu-boussard** [comment]
please update changelog

**Review by matthieu-boussard — APPROVED** [review]

**Pull request merged**